src/components/Table/index.tsx

```diff
@@ -211,7 +211,7 @@ class Table extends React.Component<TableProps, TableState> {
     }
 
     private renderHead(row: CellData[]) {
-        const cells = row.map((c, index) => <th key={index}>{c}</th>);
+        const cells = row.map((c, index) => c ?  <th key={index}>{c}</th> : <th key={index}>&nbsp;</th>);
```

Comment:
IHMO, totally wrong solution.
This is the base component which is used on many widgets. Hiding the first header cell does not correct.
If you need to hide close icon for specific component which use Table, just don't pass it to the table.

Reply:
Why do you think, that I hide the first cell? This fix made cells for full width and height of the header by adding a space character for prevent overlapping close icon on header

Reply:
`Why do you think, that I hide the first cell?`
My mistake.

Anyway, I meant the whole impact to basic component. 
Better to pass that empty space from the containers component. 

    <Table headers=['Price', .... <>&nbsp;</>] ... />

Reply:
Maybe you are right, but this solution doesn't prevent similar issues in the future unlike mine